src/parser.c

```diff
@@ -18,6 +18,9 @@
 #include <stdlib.h>
 #include <ctype.h>
 #include <assert.h>
+#if defined(__MINGW64__) || defined(__MINGW32__)
+    #include <time.h>
```

Comment:
This makes the include available only for mingw. It does not need to be available to any other system?

Reply:
I have not tried any other system. Ideally a CI or attempt to produce a release should tell you, however there's never been a release of the blackbox tools for BF since the repo was cloned from the Cleanflight repo so who knows... 😄 

See:

https://github.com/betaflight/blackbox-tools/releases

vs

https://github.com/cleanflight/blackbox-tools/releases

Today I had the need to run the tool from an old log, encountered this issue and made a fix that worked for me and shared it via this PR, I didn't spend any more time on it other than required to get the tool building and submitting the PR.